python-avocado.spec

```diff
@@ -148,6 +148,7 @@ popd
 %{__mkdir} -p %{buildroot}%{_mandir}/man1
 %{__install} -m 0644 man/avocado.1 %{buildroot}%{_mandir}/man1/avocado.1
 %{__install} -m 0644 man/avocado-rest-client.1 %{buildroot}%{_mandir}/man1/avocado-rest-client.1
+%{__install} -d -m 0755 %{buildroot}%{_sharedstatedir}/avocado/data
```

Comment:
Our spec file is "supported" for Fedora/RHEL only, and all this spec file change does is that it creates (and "packs") the directory by default on package installation.

Other distros may have to adapt this, although I doubt something such as `%{_sharedstatedir}` would be different.

Anyway, what are you proposing/requesting here?